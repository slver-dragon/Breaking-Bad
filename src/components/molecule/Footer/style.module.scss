@import "../../../assets/styles/style";

.main {
  display: flex;
  @include shadow;
  @include standarFont;
  max-width: 23.75rem;
  margin-bottom: 1.5rem;
  text-decoration: none;
  .photoWrapper {
    width: 7.75rem;
    height: 7.75rem;
    object-fit: cover;
    margin-right: 6.33%;
    overflow-y: hidden;
    border-radius: 0.25rem 0 0 0.25rem;
    .photo {
      max-width: 7.75rem;
      height: auto;
    }
  }
}
