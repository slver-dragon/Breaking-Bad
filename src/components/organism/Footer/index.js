import React from "react";
import style from "./style.module.scss";

const Footer = () => {
  return (
    <div className={style.main}>
      <p>2021</p>
      <p>@author</p>
    </div>
  )
}

export default Footer;