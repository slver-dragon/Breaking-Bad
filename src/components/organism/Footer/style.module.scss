@import "../../../assets/styles/style";

.main {
  display: flex;
  justify-content: space-between;
  .container {
    display: flex;
    justify-content: space-between;
    @include mediumFont;
    @include mainContainer;
    @include footerMain;
    line-height: 1.5rem;
    width: 100%;
  }
}
