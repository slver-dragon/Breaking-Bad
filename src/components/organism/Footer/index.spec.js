import React from "react";
import { shallow } from "enzyme";
import Footer from ".";

describe("Testing organism Footer:", () => {
  test("is rendering", () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});