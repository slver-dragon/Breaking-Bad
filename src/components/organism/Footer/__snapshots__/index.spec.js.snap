// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing organism Footer: is rendering 1`] = `
<div
  className="main"
>
  <div
    className="container"
  >
    <p>
      2021
    </p>
    <p>
      @Эдуард Пузыно
    </p>
  </div>
</div>
`;
